css/style.css

```diff
@@ -286,14 +286,92 @@ article.personDescription {
     bottom: 35px;
 }
 
- div#lightPurpleButton {
+div#lightPurpleButton {
     text-align: center;
- }
+}
 
- #sectionFive {
+#sectionFive {
     background-color: #8B48BF;
     display: flex;
     justify-content: space-between;
     align-items: center;
     padding: 40px 230px 40px 205px;
- }
\ No newline at end of file
+}
+
+#navBar {
+    width: 397px;
+}
+
+ul {
+    list-style: none;
+    padding: 0;
+}
+
+#navBarUl {
+    height: 23px;
+    display: flex;
+    justify-content: space-between;
+}
+
+a {
+    text-decoration: none;
+}
+
+.navLink {
+    position: relative;
+    font-family: 'Be Vietnam', sans-serif;
+    font-weight: 500;
+    font-size: 13px;
+    color: #242E52;
+    /* Figma especifica esta propiedad que establece espacio entre las líneas pero considero que es un error ya que esta es una lista horizontal */
+    /* line-height: 18.99px; */
+}
+
+a.navLink:hover {
+    color: rgba(36, 46, 82, 0.6);
+    cursor: pointer;
+}
+
+.verticalLink {
+    font-family: 'Be Vietnam', sans-serif;
+    font-size: 16px;
+    font-weight: 400;
+    color: #FFFFFF;
+    line-height: 23.38px;
+}
+
+#verticalList1 {
+    width: 69px;
+    height: 128px;
+}
+
+#verticalList2 {
+    width: 105px;
+    height: 93px;
+    gap: 12px;
+}
+
+input {
+    font-family: 'Be Vietnam', sans-serif;
+}
+
+input#update {
+    width: 201px;
+    height: 44px;
+    border-radius: 50px;
+    padding: 10px;
+    font-weight: 600;
+    font-size: 13.18px;
+    color: #A0A0A0;
+    line-height: 19.26;
+    background-color: #FFFFFF;
+    /* Necesario para eliminar un borde que al parecer le estaba generando el navegador */
+    border: none;
+    /* gap: 10px; lo indica figma pero no tiene sentido */
+}
+
+input#update::placeholder {
+    font-size: 13.18px;
+    line-height: 19px;
+    color: #A0A0A0;
+}
```

Comment:
**Suggestion**
It would be nice if you add a pseudoclass focus to the form in input only with the attribute color: orange. Something similar to the following example. 

![image](https://github.com/nikollquinteroch/FrontEnd-Final-Evaluation/assets/136136120/d5ccac8a-4abc-478a-8a9d-22e64354261d)

Reply:
ok!